test/train-sets/ref/automl_readable_qc.txt

```diff
@@ -1801,7 +1791,7 @@ event_sum 0
 action_sum 0
 :1
 initial_t 0
-norm normalizer 0
+norm normalizer 98.9861
```

Comment:
since we are already breaking the model, we could also remove this norm and total_weight since they are duplicate. Thoughts?

Reply:
we should move this to next PR which can break model